statsmodels/tsa/varma_process.py

```diff
@@ -464,11 +466,14 @@ class VarmaPoly(object):
 
 
     '''
+
+    nvars = deprecated_alias('nvars', 'neqs', '0.11.0')
```

Comment:
this could have a very short test (hopefully it can be added to an existing test)
I will need to be deleted again later